.github/workflows/ci-publish.yml

```diff
@@ -0,0 +1,127 @@
+        run: |
+          mkdir -p ~/.m2/
+          echo "<settingsSecurity><master>$MASTER_PASSWORD</master></settingsSecurity>" > ~/.m2/settings-security.xml
+          echo "<settings><servers><server><id>ossrh</id><username>$OSSRH_USERNAME</username><password>$OSSRH_PASSWORD</password></server></servers></settings>" > ~/.m2/settings.xml
+        env:
+          MASTER_PASSWORD: ${{ secrets.MASTER_PASSWORD }}
+          OSSRH_USERNAME: ${{ secrets.OSSRH_USERNAME }}
+          OSSRH_PASSWORD: ${{ secrets.OSSRH_PASSWORD }}
+      # TODO: the step "Make JanusGraph JAR artifacts" should also create signature key for full JanusGraph bundle but for some
+      # reason it doesn't sign full distribution (only standard distribution).
+      # After this is fixed we can delete the step "Sign JanusGraph full distribution artifact".
+      - name: Make JanusGraph JAR artifacts
+        run: mvn clean install -Pjanusgraph-release -DskipTests=true
+      - name: Print content of the generated target directory for debugging purposes
+        run: echo $(ls janusgraph-dist/target/)
+      - name: Deploy JanusGraph JAR artifacts into staging repository
+        run: mvn deploy -Pjanusgraph-release -DskipTests=true
+      - name: Sign JanusGraph full distribution artifact
+        run: gpg --armor --detach-sign --yes janusgraph-dist/target/${JG_FULL_DIST_NAME}.zip
+      - name: Install requirements to build documentation
+        run: pip3 install -r requirements.txt
+      - name: Build documentation
+        run: mkdocs build
+      - name: Pack documentation into zip
+        run: |
+          mv site ${JG_DOC_DIST_NAME}
+          zip -r ${JG_DOC_DIST_NAME}.zip ${JG_DOC_DIST_NAME}
+      - name: Sign documentation artifact
+        run: gpg --armor --detach-sign ${JG_DOC_DIST_NAME}.zip
+      - name: Check distribution signature
+        run: gpg --verify janusgraph-dist/target/${JG_DIST_NAME}.zip.asc janusgraph-dist/target/${JG_DIST_NAME}.zip
+      - name: Check full distribution signature
+        run: gpg --verify janusgraph-dist/target/${JG_FULL_DIST_NAME}.zip.asc janusgraph-dist/target/${JG_FULL_DIST_NAME}.zip
+      - name: Check documentation signature
+        run: gpg --verify ${JG_DOC_DIST_NAME}.zip.asc ${JG_DOC_DIST_NAME}.zip
+      - uses: ncipollo/release-action@v1
+        with:
+          allowUpdates: true
+          artifactErrorsFailBuild: true
+          draft: true
+          tag: "${{ env.RELEASE_TAG }}"
+          artifacts: "KEYS,${{ env.JG_DOC_DIST_NAME }}.zip.asc,${{ env.JG_DOC_DIST_NAME }}.zip,janusgraph-dist/target/${{ env.JG_DIST_NAME }}.zip.asc,janusgraph-dist/target/${{ env.JG_DIST_NAME }}.zip,janusgraph-dist/target/${{ env.JG_FULL_DIST_NAME }}.zip.asc,janusgraph-dist/target/${{ env.JG_FULL_DIST_NAME }}.zip"
+          name: "${{ env.JG_VER }}"
+          omitBodyDuringUpdate: true
+          omitNameDuringUpdate: true
+          omitPrereleaseDuringUpdate: true
+          owner: janusgraph-bot
```

Comment:
I think `janusgraph-bot` should work as an owner here but didn't verify it yet. I used my nickname `porunov` for testing in my fork repository. I requested collaboration access to my fork here: https://github.com/porunov/janusgraph/ 
@mbrukman are you the owner of `janusgraph-bot` account? If so, will you be able to accept collaboration request to https://github.com/porunov/janusgraph/ for [`janusgraph-bot`](https://github.com/janusgraph-bot)? If you lost the invite I can re-send it again.